src/renderer/css/index.scss

```diff
@@ -0,0 +1,219 @@
+    overflow: hidden;
+    height: 40px;
+    background-color: #efefef;
+    width: 100%;
+  }
+
+  .topBar>.topBar-bg.unfocused {
+    opacity: 0.4;
+  }
+
+  .topBar.darkMode>.topBar-bg {
+    background-color: #2e2e2e;
+  }
+
+  .topBar .three-dot-menu {
+    font-size: 20px;
+    line-height: 20px;
+    height: 36px;
+    float: left;
+    margin-left: 4px;
+    padding-top: 5px;
+    border: none;
+    flex: 0 0 40px;
+    z-index: 9;
+    color: rgba(61,60,64,0.7);
+    -webkit-app-region: no-drag;
+    background-color: transparent;
+  }
+
+  .topBar .three-dot-menu i.icon-dots-vertical {
+    border-radius: 100px;
+    padding: 2px;
+    height: 28px;
+    width: 28px;
+  }
+
+  .topBar .three-dot-menu:focus {
+    outline: none;
+  }
+
+  .topBar .three-dot-menu:hover i.icon-dots-vertical, .topBar .three-dot-menu:focus i.icon-dots-vertical, .topBar .three-dot-menu:active i.icon-dots-vertical {
+    outline: none;
+    background-color: #c8c8c8;
+  }
+
+  .topBar.darkMode .three-dot-menu:hover i.icon-dots-vertical, .topBar.darkMode .three-dot-menu:focus i.icon-dots-vertical, .topBar.darkMode .three-dot-menu:active i.icon-dots-vertical {
+    background-color: #383A3F;
+  }
+
+  .topBar.macOS .three-dot-menu:hover i.icon-dots-vertical, .topBar.macOS .three-dot-menu:focus i.icon-dots-vertical, .topBar.macOS .three-dot-menu:active i.icon-dots-vertical {
+    background: none !important;
+  }
+
+  .topBar.macOS .three-dot-menu {
+    flex-basis: 80px;
+    -webkit-app-region: drag;
+  }
+
+  .topBar.macOS.fullScreen .three-dot-menu {
+    flex-basis: 0px;
+  }
+
+  .topBar.macOS .three-dot-menu>i.icon-dots-vertical {
+    display: none;
+  }
+
+  .topBar.darkMode .three-dot-menu {
+    color: rgba(243,243,243,0.7);
+  }
+
+  .topBar.darkMode .title-bar-btns {
+    color: rgba(243,243,243,0.7);
+  }
+
+  .topBar .title-bar-btns {
+    position: relative;
+    line-height: 40px;
+    height: 40px;
+    z-index: 9;
+    color: rgba(61,60,64,0.7);
+    -webkit-app-region: no-drag;
+    display: grid;
+    grid-template-columns: repeat(3, 46px);
+  }
+
+  .topBar .title-bar-btns>.button {
+    grid-row: 1 / span 1;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+    user-select: none;
+  }
+
+  .topBar.darkMode .title-bar-btns>.button:hover {
+    background: rgba(255,255,255,0.1);
+  }
+
+  .topBar.darkMode .title-bar-btns>.button:active {
+    background: rgba(255,255,255,0.2);
+  }
+
+  .topBar .title-bar-btns>.button:hover {
+    background: rgba(0,0,0,0.1);
+  }
+
+  .topBar .title-bar-btns>.button:active {
+    background: rgba(0,0,0,0.2);
+  }
+
+  .topBar .title-bar-btns>.close-button:hover {
+    background: #E81123 !important;
+  }
+
+  .topBar .title-bar-btns>.close-button:hover>img {
+    filter: invert(100%);
+    -webkit-filter: invert(100%);
+    opacity: 1;
+  }
+
+  .topBar .title-bar-btns>.close-button:active {
+    background: #f1707a !important;
```

Comment:
I would investigate why these importants are needed. IMO the styles need some refactoring